src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -14,40 +14,51 @@ using namespace sf;
 using namespace ssvs;
 using namespace ssvuj;
 
-namespace hg
-{
+namespace hg{
+
+    void HexagonGame::redefineLuaFunctions(){
+        try{
+            lua.executeCode("local open = io.open; io.open = function(filename) return open(filename, \"r\"); end");
+        }
+        catch(...){
```

Comment:
```suggestion
        catch(...)
        {
```

Reply:
blame my IDE i guess

Reply:
Should be able to change that in the IDE's settings. Which IDE do ya use, again?

Reply:
I know I can but i honestly don't want to since i don't prefer this style of writing.
My IDE somehow supports clang-format but for some unknown reason it doesn't work so i have to do this manualy throug therminal.

Reply:
It supposed to format before commit, but it doesn't.